pagopa/io-eudiw-app
# fix [EUDIW-56]: add biometric identification while restart, biometric configurat…

## Short description
Add biometric identification while restart application.
Biometric configuration persisted after closing application.

## List of changes proposed in this pull request
- Feature A
- Feature B

## How to Test
Step 1: Complete First Application Onboarding

- Open the application for the first time.
- Follow the onboarding process and set up biometric configuration for identification.
- Complete all required steps in the onboarding process.
- Verify that the biometric configuration is successfully set up.

Step 2: Close the Application

- Close the application completely.

Step 3: Verify Identification Modal on Start

- Reopen the application.
- Verify that the application shows an identification modal requiring biometric authentication.

Step 4: Verify Biometric Authentication for Operations

- Close the application again.
- Reopen the application and perform an operation such as adding a document.
- Verify that the application requests biometric identification before allowing the operation.

Expected Results
After the first onboarding, the application should prompt for biometric identification on startup.
The application should request biometric identification to allow operations like adding a document after it has been closed and reopened.
